actions/setup_and_build/action.yml

```diff
@@ -1,29 +1,25 @@
-
 name: 'Setup and build'
 description: 'Setup and build Session Desktop'
+inputs:
+  cache_suffix:
+    description: 'the package we are currently building (used as key for the cached node_modules)'
+    required: true
+
 runs:
   using: 'composite'
   steps:
     - name: Install node
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
         node-version-file: '.nvmrc'
 
-    - uses: actions/setup-python@v4
+    - uses: actions/setup-python@v5
       with:
         python-version: '3.11'
```

Comment:
Will this use the latest python 3.11.X ? If it is installing 3.11.0 we might want to tweak that for security?

Reply:
yeah latest 3.11.X. We can't upgrade for now, will need to fix the scripts first

Reply:
Cool cool